contracts/views/ConvexView.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+import "../actions/convex/helpers/ConvexHelper.sol";
+
+contract ConvexView is ConvexHelper {
```

Comment:
It's a view but i see nothing

Reply:
will update as soon as we sync with front-end